lib/gprocess.c

```diff
@@ -1543,6 +1544,7 @@ g_process_finish(void)
   */
 #endif
 
+  free(process_opts.argv_orig);
```

Comment:
There's a comment right above this new line that explains why we don't want to free that line. I am more than happy to prove that this comment is already stale and we can actually perform this free operation, but please remove the comment along with it.

Reply:
Said comment is the reason, why I only freed this variable, and not `**environ`. However the fact that somehow I placed the free outside the if clause is proof enough that I should have drank at least two more cups of coffee yesterday.
I would be happy to investigate this leak further, but given the circumstances, I am not confident enough to still include this in this merge request, as the other fix is instrumental to the fuzzing project, so I do not want to delay that one.
Sorry for the blunder.